.github/workflows/run_tests.yaml

```diff
@@ -37,8 +37,9 @@ jobs:
         run: |
           echo '${{ inputs.rock-metas }}' | jq -c '.[] | select(.arch == "amd64") ' | while read i; do
             name=$(echo "$i" | jq -r '.name' | tr '[:lower:]' '[:upper:]' | tr '-' '_')
+            version=$(echo "$i" | jq -r '.version' | sed 's/^v\?/v/' | tr '[:lower:]' '[:upper:]' | tr '.' '_')
```

Comment:
Small suggestion: avoid adding an extra lowercase 'v' in case the version string already starts with a _capital_ 'V': `sed 's/^[vV]\?/v/'`

(Just checked and `rockcraft` does NOT prohibit this for the `version` field in `rockcraft.yaml`, only the `name` field)

Reply:
I'd also suggest `tr '.\-+' '_'` at the end in case we have rockfiles with full SemVer `-pre-release` or `+build` tags.

```bash
$ echo "1.2.3-rc0+123" | sed 's/^[vV]\?/v/' | tr '[:lower:]' '[:upper:]' | tr '.\-+' '_'
V1_2_3_RC0_123
```